ModelBundle/Repository/NodeRepository.php

```diff
@@ -733,6 +751,29 @@ public function findByParentAndRoutePattern($parentId, $routePattern, $nodeId, $
     }
 
     /**
+     * @param string $parentId
+     * @param int    $order
+     * @param string $nodeId
+     * @param string $siteId
+     *
+     * @return bool
+     */
+    public function findByParentAndOrderAndNotNode($parentId, $order, $nodeId, $siteId)
```

Comment:
If you put findBy you expect to get a node as output. I would suggest to start with hasNode or something like that